docs/docs-graphql/schema.graphqls

```diff
@@ -168,7 +168,7 @@ type Container {
   ): Container!
 
   """
-  Writes the container as an OCI tarball to the destination file path on the host.
+  Writes the container as an OCI tarball to the destination file path on the current runtime container spawned by Dagger engine.
```

Comment:
I'm not sure it's necessary to update this file. AFAIK it is generated from the core schema. This might also be why the tests are failing. I'd like other reviewers to confirm this though.

Reply:
When changes are made to the schema, you need a `dagger call` to regenerate docs and sdks.

https://github.com/dagger/dagger/blob/447c8a416f554e55aa2c9f0925f83d381b90b8cc/.dagger/README.md?plain=1#L81

https://github.com/dagger/dagger/blob/447c8a416f554e55aa2c9f0925f83d381b90b8cc/.dagger/README.md?plain=1#L116-L118